android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/Account.kt

```diff
@@ -0,0 +1,42 @@
+package com.mytiki.sdk.capture.receipt.capacitor
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import com.getcapacitor.JSObject
+import com.microblink.digital.PasswordCredentials
+import com.mytiki.sdk.capture.receipt.capacitor.req.ReqAccount
+import com.mytiki.sdk.capture.receipt.capacitor.rsp.RspAccount
+import com.mytiki.sdk.capture.receipt.capacitor.rsp.RspAccountList
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.async
+
+class Account(
+    val accountCommon: AccountCommon,
+    val username: String,
+    val password: String? = null,
+    var isVerified: Boolean? = null
+) {
+    fun toRsp(): JSObject = JSObject.fromJSONObject(RspAccount(this).toJson())
+    companion object{
+        fun fromReq(data: JSObject): Account {
+            val req = ReqAccount(data)
+            return Account(req.accountCommon, req.username, req.password, req.isVerified)
+        }
+
+        fun fromMbLinking(mbAccount: com.microblink.linking.Account): Account{
+            val accountType = AccountCommon.fromString(RetailerEnum.fromValue(mbAccount.retailerId).toString())
+            val username = mbAccount.credentials.username()
+            return Account(accountType, username)
+        }
+
+        fun fromMbEmail(mbAccount:  PasswordCredentials): Account{
+            val accountType = AccountCommon.fromString(mbAccount.provider().name)
+            val username = mbAccount.username()
+            return Account(accountType, username)
+        }
+
+        @RequiresApi(Build.VERSION_CODES.O_MR1)
```

Comment:
the android studio asked for this annotation, because the min api level of the project is 22 but the call asks for api 27 as minimum, that annotation is in other places of the code